charts/argo-cd/templates/argocd-configs/argocd-secret.yaml

```diff
@@ -34,9 +34,9 @@ data:
   {{- with .Values.configs.secret.gogsSecret }}
   webhook.gogs.secret: {{ . | b64enc }}
   {{- end }}
-  {{- with .Values.configs.secret.azureDevops }}
-  webhook.azuredevops.username: {{ .username }}
-  webhook.azuredevops.password: {{ .password | b64enc }}
+  {{- if and (.Values.configs.secret.azureDevops.username .Values.configs.secret.azureDevops.password) }}
+  webhook.azuredevops.username: {{ .Values.configs.secret.azureDevops.username }}
```

Comment:
We also need b64enc here 🙃

Reply:
Ah~~~~~ will fix 😇 💣

Reply:
I fixed in https://github.com/argoproj/argo-helm/pull/2443/commits/ccb41f08b502ec670682ab695f7ec43523869b58 . 🏥 💉